### DISCUSSION TITLE
1주차 회고

### DISCUSSION THREAD

**Discussion by JiinSeok** [team-final/albaform#48] [General]
[회고란?](https://github.com/codeit-bootcamp-frontend/7-Sprint-Mission/discussions/293)

지난 주 팀 결성부터 이번주 현 시점까지 Team-Final을 돌아보며, Keep, Problem, Try를 적어주세요!

**Comment by JiinSeok** [comment]
# KEEP

**Reply by gayeong0916** [reply]
코어 타임이 아닌 시간임에도 불구하고 소통이 빨라서 좋았다.
처음 접해서 잘 알지 못하고 익숙하지 않은 것들에 대해 팀원들에게 여러 번 여쭤보았는데, 친절하게 가르쳐 주셔서 좋았다.

**Reply by JiinSeok** [reply]
- @clyde-yoon 님의 적극적인 참여로 첫주 안에 프로젝트 파악이 잘 시작되어서 감사하다!

**Reply by JiinSeok** [reply]
어려운 내용임에도 팀원들이 포기하지 않고 도움을 구하고 배움으로써 협업 코드의 유지보수성을 높이는 git flow를 적용할 수 있었다.
스터디처럼 주제 공지하고 자료 발표하고 진행했으면 쉬웠을까 아쉬움이 있지만, git flow 자체가 처음부터 이해하거나 필요성을 느끼기엔 어려운 주제가 아닐까 싶다.

**Reply by JiinSeok** [reply]
코어타임 제도! 꾸준히 프로젝트가 진행되게 하고, 사람들을 모이게 해주어 초반 분위기에 도움이 되었다

**Reply by JuhyeokC** [reply]
- 코어타임+- 으로 모각코 참여도가 높아서 좋다

**Reply by engdawn** [reply]
문제가 발생하면 빠르게 상황 공유가 되고 소통이 원활합니다.

**Reply by clyde-yoon** [reply]
우리팀 하면 역시 소통이 제일 최고인것같다!

**Comment by JiinSeok** [comment]
# PROBLEM

**Reply by gayeong0916** [reply]
이슈를 통해 브랜치를 생성하고 fetch & rebase를 쓰는 과정이 익숙하지 않아서 프로젝트의 1주일 정도를 소비하였다.

**Reply by JiinSeok** [reply]
기획 파악을 첫 이슈에 포함시키지 않은 아쉬움!
- 내용 방면에 대한 논의를 놓치다 보니 깔끔한 개발환경이 만들어졌음에도 프로젝트가 완전히 시작되지 않은 듯 붕 뜬 느낌이 있었던 것 같다. 팀원들의 첫 에너지를 프로젝트 인볼브(ㅎㅎ)에 꼭 써야 한다는 걸 깨달았다!!!!
- 팀미팅에서 개발 방법인 워크플로우와 기술스택을 합의하는 데에 집중했는데, 개발 내용을 정리하고 합의하는 것은 많이 부족했다. 팀장인 내가 해오는 것이 팀원들을 위한 일이라고 생각했는데 오판이었다. 프로젝트 이슈는 개발 그 자체에 대한 것이어야 한다는 선입견이 있었는데, 그건 PM/기획자/개발관리자 등이 그 외를 처리하기 때문이었다. 내가 하려면 내가 PM을 해야 했다. 
- 다시 생각하니 맡은 이슈와 그 이슈가 속한 프로젝트를 주도적으로 파악하는 것은 실무 개발자로서 너무나 중요한 경험이고 능력이었다. 비개발자로서 맡던 역할에 이입해 실무 개발자라면 어떻게 기획을 봐야 할까 생각지 못한 것 같아 팀원들에게 미안하다.

**Reply by JuhyeokC** [reply]
- 깃 플로우와 깃헙 사용에 익숙해지면 좋을 것 같다
- 코어타임에 팀미팅 시간, 회의 시간이 너무 길다.

**Reply by engdawn** [reply]
깃이 어려웠다.

**Reply by clyde-yoon** [reply]
개발에서는 너무나도 많은 기술과 정보들이 존재하는데 배움이 앞서, 처음접하는 방식에 대한 학습이 안일했었다. 그럼에도 도움 많이 주신 팀원들에게 너무나도감사합니다.

**Comment by JiinSeok** [comment]
# TRY

**Reply by gayeong0916** [reply]
다른 팀원들에게 물어보면서 시간을 뺏기보다 git의 작동 원리에 대해 혼자 더 공부하는 시간을 가지는게 좋을 거 같다.

**Reply by JiinSeok** [reply]
앞으로 팀미팅을 스터디를 운영한다는 마음으로, 다음 팀미팅의 주제를 정하고 각자 고민해와서 결정할 수 있게 운영하고 싶다.
개발자들에게 익숙한 방식으로 모두가 더 적극적으로 참여할 수 있을 것 같다.
1. 프로젝트 기획 파악처럼 같이 해야 빠르고 정확한 주제에 적합하다.
2. 구현, 협업 방식에 적용하고 싶은 지식에 대해, 스터디 주제 정하듯 사전합의를 하면 팀이 준비된 상태에서 더 빠르게 받아들일 수 있다.

**Reply by JiinSeok** [reply]
팀미팅에 할 말이 있으면 이 디스커션 탭에 글을 올려두는 것도 좋을 것 같다. 
1. 글로 정리해야 하므로 스스로 어떤 것이 문제인지 정확히 규정하고 공유할 수 있다. 필요로 하는 도움, 의견을 정확히 할 수 있다.
2. 팀미팅 시간을 단축할 수 있다.

**Reply by JuhyeokC** [reply]
- 팀미팅, 회의 시간에 핵심 내용을 중심으로 이야기 하고 PR 리뷰와 개발 작업에 집중할 수 있으면 좋겠다.
- 깃 플로우와 깃헙 사용을 적극적으로 이용하자, 도와줄 수 있는 팀원이 있다면 익숙해질 때 까지 계속 물어보자. 그 사람도 복기해보며 도움된다.

**Reply by engdawn** [reply]
깃 반복 사용을 통한 숙달

**Reply by clyde-yoon** [reply]
빠르게 그리고, 신속하고 확실한 정보전달 및 의견전달을 노력하겠습니다.
